model/src/components/types.ts

```diff
@@ -264,6 +265,15 @@ export interface RadiosFieldComponent extends ListFieldBase {
 
 export interface SelectFieldComponent extends ListFieldBase {
   type: "SelectField";
+  options: {
+    type?: string;
+    hideTitle?: boolean;
+    required?: boolean;
+    optionalText?: boolean;
+    classes?: string;
+    bold?: boolean;
+    autocomplete?: string;
+  };
```

Comment:
this looks like a duplicate of ListFieldBase options + an additional option, so you can use Type["nested"] so you don't have to dupe
```suggestion
  options: ListFieldBase["options"] & { autocomplete?: string };
```

Reply:
👍 awesome, will update

Reply:
Done